src/components/toggle/toggle-option/toggle-option.scss

```diff
@@ -49,7 +49,7 @@ input[type='radio'] {
   border-radius: var(--sbb-toggle-option-border-radius);
   color: var(--sbb-toggle-option-color);
 
-  :host(:not([data-icon-only])) & {
+  :host([data-slot-names~='unnamed']:where([data-slot-names~='icon'], [icon-name])) & {
```

Comment:
with the new solution there is a glitch for the icon only variant. see here: https://lyne-components-preview-pr2290.app.sbb.ch/?path=/story/components-sbb-toggle-sbb-toggle--icons-only

Reply:
here even more noticeable in normal mode: https://lyne-components-preview-pr2290.app.sbb.ch/?path=/story/components-sbb-toggle-sbb-toggle--dynamic-width

Reply:
Thanks for noticing. My logic for detecting whether something is slotted in a slot was too naive.
I fixed it now.